corehq/apps/users/templates/users/edit_web_user.html

```diff
@@ -5,6 +5,33 @@
 {% load i18n %}
 
 {% block page_content %}
+  {% if request|toggle_enabled:"ENTERPRISE_SSO" and has_untrusted_identity_provider %}
+    <div class="alert alert-setting">
+      <p class="lead">
+        <i class="fa fa-lock"></i>
+        {% blocktrans with couch_user.human_friendly_name as friendly_name %}
+          The SSO Identity Provider for {{ friendly_name }}, "{{ idp_name }}",
+          is not trusted by this project space.
+        {% endblocktrans %}
+      </p>
+      <p>
+        {% blocktrans %}
+          Click Trust Provider below to restore access for this user.
+          <a href="#">Learn more</a> about SSO on CommCare HQ.
+        {% endblocktrans %}
+      </p>
+      <form class="form form" name="trust_identity_provider" method="post">
+        {% csrf_token %}
+        <input type="hidden" name="form_type" value="trust-identity-provider" />
```

Comment:
we use `form_type` to differentiate the different form `POST`s on this view. This is not the only form.

Reply:
I think there are a lot of benefits to handling forms in separate views, but that's probably out of scope here. However, to just differentiate between forms, the convention seems to be reading the name from the submit button. [This stackoverflow thread](https://stackoverflow.com/a/1395866) has some decent discussion of the options. The submit button just seems like a cleaner solution than a hidden field, but I guess I'm nitpicking here.

Reply:
one could argue that there are lots of benefits of processing forms in the same view—especially ensuring that permissions stay consistent. since we followed a different convention already for this view, I'm going to stick with the same convention.